src/Storefront/Framework/Routing/Router.php

```diff
@@ -194,8 +194,7 @@ private function getBasePath(): string
 
     private function isStorefrontRoute(string $name): bool
     {
-        return str_starts_with($name, 'frontend.')
-            || str_starts_with($name, 'widgets.')
-            || str_starts_with($name, 'payment.');
+        $route = $this->getRouteCollection()->get($name);
+        return \in_array(StorefrontRouteScope::ID, $route?->getDefault(PlatformRequest::ATTRIBUTE_ROUTE_SCOPE) ?? [], true);
```

Comment:
empty line is missing before the return, that's why the linting job is failing 
```suggestion

        return \in_array(StorefrontRouteScope::ID, $route?->getDefault(PlatformRequest::ATTRIBUTE_ROUTE_SCOPE) ?? [], true);
```

Reply:
add empty line